tutorials/Land/Soil/Water/equilibrium_test.jl

```diff
@@ -45,48 +45,48 @@ using ClimateMachine.BalanceLaws:
 
 
 # - define the float type desired (`Float64` or `Float32`)
-FT = Float64;
-
+FT = Float64
 # - initialize ClimateMachine for CPU
-ClimateMachine.init(; disable_gpu = true);
-
-
+ClimateMachine.init(; disable_gpu = true)
 # # Set up the soil model
 
-# We want to solveRichard's equation alone, i.e. do not solve
-# the heat equation. The default is a constant temperature, but this
-# only affects Richard's equation if one chooses a temperature dependent
-# viscosity of water (see below).
+# We want to solve Richard's equation alone, without simultaneously
+# solving the heat equation. One can choose to essentially ignore temperature
+# (the default), or to choose a prescribed temperature function with which to drive
+# Richard's equation (via the temperature dependence of viscosity, or via freeze/thaw).
+# Here we ignore the effects of temperature and freezing and thawing. 
 
 soil_heat_model = PrescribedTemperatureModel{FT}()
 
 # Define the porosity, Ksat, and specific storage values for the soil. Note
-# that all values must be givne in mks.
-soil_param_functions = SoilParamFunctions(
+# that all values must be given in mks units. The soil parameters chosen
+# roughly correspond to Yolo light clay.
+soil_param_functions = SoilParamFunctions{FT}(
     porosity = 0.495,
     Ksat = 0.0443 / (3600 * 100),
     S_s = 1e-3,
 )
 
 # Define the boundary conditions. The user can specify two conditions,
 # either at the top or at the bottom, and they can either be Dirichlet
-# (on `ϑ_l`) or Neumann (on `-K∇h`). Note that fluxes are supplied as
+# (on `ϑ_l`) or Neumann (on `K∇h`). Note that fluxes are supplied as
 # scalars, inside the code they are multiplied by ẑ. The two conditions
-# not supplied must be set to `nothing`.
-
-surface_flux = (aux, t) -> FT(0.0)
-bottom_flux = (aux, t) -> FT(0.0)
+# not supplied must be set to `nothing`. 
+zero_value = FT(0.0)
+surface_flux = (aux, t) -> zero_value
+bottom_flux = (aux, t) -> zero_value
```

Comment:
```suggestion
surface_flux = (aux, t) -> eltype(aux)(0)
bottom_flux = (aux, t) -> eltype(aux)(0)
```

Reply:
hey Charlie - Im not sure what happened, but this file looks different in the branch than what you reviewed. But, the changes you made apply to the tutorials I am working on (somehow that file made its way into this at some point), so I will make them there.

Reply:
I can actually make them here too. will do that!